src/angular/controllers/ctrl-widget-player.test.js

```diff
@@ -0,0 +1,894 @@
+describe('WidgetPlayerCtrl', () => {
```

Comment:
Since the widget player page has been completely rewritten in React, this file is no longer necessary. I'll make a separate commit to remove it.